app/models/client.rb

```diff
@@ -0,0 +1,6 @@
+class Client < ApplicationRecord
+  validates :name, :email, presence: true, length: { minimum: 3 }
```

Comment:
should there not be some email format validation, look into it please.

Reply:
Have used devise form here and it validates for email formate.
should I used it ?

Reply:
i guess we had a discussion on it, add some backend validation here, you only have front end one for now.

Reply:
have added backend validation.